libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -317,6 +317,27 @@ public static Task<TeamsChannelAccount> GetMemberAsync(ITurnContext turnContext,
             return new Tuple<ConversationReference, string>(conversationReference, newActivityId);
         }
 
+        /// <summary>
+        /// Sends a notification to meeting participants. This functionality is available only in teams meeting scoped conversations. 
+        /// </summary>
+        /// <param name="turnContext">Turn context.</param>
+        /// <param name="notification">The notification to send to Teams.</param>
+        /// <param name="meetingId">The id of the Teams meeting. TeamsChannelData.Meeting.Id will be used if none provided.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        /// <remarks>InvalidOperationException will be thrown if meetingId or notification have not been
```

Comment:
Since this method is a public api, does it makes sense to have `InvalidOperationException` be "linkable" like with the `<see cref="TeamsMeetingNotificationRecipientFailureInfo"/>